modules/i18n/src/models/namespaces/branding-ns.ts

```diff
@@ -168,4 +156,46 @@ export interface BrandingNS {
         "password-reset": string;
         "password-reset-success": string;
     };
+    ai: {
+        banner: {
+            full: {
+                heading: string;
+                subHeading: string;
+                button: string;
+            };
+            input: {
+                heading: string;
+                subHeading: string;
+                placeholder: string;
+                button: string;
+            };
+            collapsed: {
+                heading: string;
+                subHeading: string;
+                button: string;
+            };
+        };
+        screens: {
+            loading: {
+                heading: string;
+                didYouKnow: string;
+                facts: {
+                    0: string;
```

Comment:
It'd be better if we could use string keys instead of numeric keys for enhanced readability, if time permits 😉

Reply:
we will revisit this when the loginflow AI comes in